### ISSUE TITLE
jekyll returns error

### ISSUE THREAD

**Issue by ksuenaga** [kuis-isle3sw/IoPLMaterials#92]
`bundle exec jekyll s` returns the following.  Maybe [my fix](https://github.com/kuis-isle3sw/IoPLMaterials/commit/13abf5c1fefb369ec66ad2ba65759c543b37b62e) is not working.  I'll take care of this later.
```
  Liquid Exception: Invalid syntax for include tag. File contains invalid characters or sequences: ../miniml-interpreter/lib/syntax.ml Valid syntax: {% include_relative file.ext param='value' param2='value' %} in textbook/chap03-2.md
jekyll 3.9.0 | Error:  Invalid syntax for include tag. File contains invalid characters or sequences:

  ../miniml-interpreter/lib/syntax.ml

Valid syntax:

  {% include_relative file.ext param='value' param2='value' %}


/Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/tags/include.rb:67:in `validate_file_name': Invalid syntax for include tag. File contains invalid characters or sequences: (ArgumentError)

  ../miniml-interpreter/lib/syntax.ml

Valid syntax:

  {% include_relative file.ext param='value' param2='value' %}

	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/tags/include.rb:128:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/block.rb:17:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/tags/highlight.rb:34:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-commonmark-ghpages-0.1.6/lib/jekyll-commonmark-ghpages.rb:100:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:82:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:30:in `block (2 levels) in render!'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:42:in `measure_bytes'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:29:in `block in render!'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:49:in `measure_time'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/liquid_renderer/file.rb:28:in `render!'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:126:in `render_liquid'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:79:in `render_document'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:62:in `run'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/site.rb:479:in `render_regenerated'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/site.rb:472:in `block in render_pages'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/site.rb:471:in `each'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/site.rb:471:in `render_pages'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/site.rb:192:in `render'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/site.rb:71:in `process'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/command.rb:28:in `process_site'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:65:in `build'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:36:in `process'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `block in start'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `each'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `start'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/gems/jekyll-3.9.0/exe/jekyll:15:in `<top (required)>'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/bin/jekyll:23:in `load'
	from /Users/ksuenaga/work/IoPL/IoPLMaterials/vendor/bundle/ruby/3.0.0/bin/jekyll:23:in `<top (required)>'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/cli/exec.rb:63:in `load'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/cli/exec.rb:63:in `kernel_load'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/cli/exec.rb:28:in `run'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/cli.rb:497:in `exec'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/cli.rb:30:in `dispatch'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/cli.rb:24:in `start'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.3/libexec/bundle:49:in `block in <top (required)>'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/3.0.0/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bundler-2.2.3/libexec/bundle:37:in `<top (required)>'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/bin/bundle:23:in `load'
	from /Users/ksuenaga/.rbenv/versions/3.0.0/bin/bundle:23:in `<main>'
```

**Comment by ksuenaga** [comment]
We need to fix the references to the interpreter code in the textbook.

**Comment by ksuenaga** [comment]
Fixed in #93
